src/components/example/ECard.tsx

```
@@ -0,0 +1,57 @@
+    SelectValue,
+} from "@/components/ui/select"
+
+export function ECard() {
+    return (
+        <Card className="w-[350px]">
+            <CardHeader>
+                <CardTitle>Test Card</CardTitle>
+                <CardDescription>Deploy your new project in one-click.</CardDescription>
+            </CardHeader>
+            <CardContent>
+                <form>
```

ESLint

[error] Delete `········`

(prettier/prettier)